### DISCUSSION TITLE
[Announcement] Tailwind CSS v1.5.0

### DISCUSSION THREAD

**Discussion by adamwathan** [tailwindlabs/tailwindcss#2033] [Show and tell]
We just released Tailwind CSS v1.5.0, with a handful of a cool new features:

- Support for creating `variants` of component classes
- Responsive variants for the `container` class
- New `focus-visible` variant
- New `checked` variant

Read more about it in the [release notes](https://github.com/tailwindcss/tailwindcss/releases/tag/v1.5.0)! 🥳

**Comment by yakushev7** [comment]
Good
